src/InputReader.java

```diff
@@ -0,0 +1,42 @@
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class InputReader {
+    private Set<String> plusContain = new HashSet<>();
+    private Set<String> minusContain = new HashSet<>();
+    private Set<String> withOutSign = new HashSet<>();
+
+    public void categorizeInput(String[] query) {
```

Comment:
**Method visibility**
Methods which doesn't use in other classes should have private visibility.